src/Symfony/Component/ExpressionLanguage/Parser.php

```diff
@@ -330,7 +330,7 @@ public function parsePostfixExpression($node)
                     throw new SyntaxError('Expected name', $token->cursor);
                 }
 
-                $arg = new Node\ConstantNode($token->value);
+                $arg = new Node\NameNode($token->value);
```

Comment:
Using `NameNode` instead of `ConstantNode` for representing `GetAttrNode::PROPERTY_CALL` and `GetAttrNode::METHOD_CALL` (but keep `ConstantNode`for `GetAttrNode::ARRAY_CALL`), the AST is more correct.
This is enlighten by the "unescaping" ([this trim](https://github.com/symfony/symfony/pull/19060/files#diff-ba72ffdc5e1ef963bfb7072610580259L102)) that was required otherwise when dumping a `GetAttrNode`.